test/hotspot/jtreg/compiler/loopopts/TestDeepGraphVerifyIterativeGVN.java

```diff
@@ -27,12 +27,14 @@
  * @requires vm.debug == true & vm.flavor == "server"
  * @summary Test which causes a stack overflow segmentation fault with -XX:VerifyIterativeGVN=1 due to a too deep recursion in Node::verify_recur().
  *
- * @run main/othervm/timeout=600 -Xcomp -XX:VerifyIterativeGVN=1 -XX:CompileCommand=compileonly,compiler.loopopts.TestDeepGraphVerifyIterativeGVN::*
+ * @run main/othervm/timeout=600 -Xcomp -XX:VerifyIterativeGVN=1 -XX:CompileCommand=compileonly,compiler.loopopts.TestDeepGraphVerifyIterativeGVN::* -XX:CompileCommand=memlimit,TestFindNode::test,0
```

Comment:
Why you list TestsFindNode here?
It is mistake in JDK-8331295 Description referencing TestsFindNode failure output. I added comment in JBS with correct TestDeepGraphVerifyIterativeGVN failure output.